cmd/abapEnvironmentCreateSystem.go

```diff
@@ -129,15 +141,53 @@ func generateManifestYAML(config *abapEnvironmentCreateSystemOptions) ([]byte, e
 	return manifestYAML, nil
 }
 
+func checkManifestParameters(config *abapEnvironmentCreateSystemOptions) (err error) {
+
+	//Only checks for correct parameter specification if CF Service "abap-oem" is selected
+	if config.CfService == "abap-oem" {
+
+		if config.AbapSystemParentSaaSAppname == "" && config.AbapSystemParentServiceLabel == "" {
```

Comment:
From the linked github page in the issue, it seems that those parameters are not mandatory. @peterpersiel did you test it somehow?

Reply:
Docu is stating: `Either parent_service_label or parent_saas_appname must be set - depending on who created the oem-instance`

Reply:
Yes, unfortunately one can only create an abap system with installed add-on while providing one of these values. There is no validation of "parent_service_label or parent_saas_appname must be set" during system creation (regex-based). But later on when the system provisioning started it will fail with such an error if not one of these values is provided.